Classes/Controller/FrontEndEditorController.php

```diff
@@ -89,4 +89,13 @@ public function createAction(Tea $tea): ResponseInterface
 
         return $this->redirect('index');
     }
+
+    public function deleteAction(Tea $tea): ResponseInterface
+    {
+        $this->checkIfUserIsOwner($tea);
+
+        $this->teaRepository->remove($tea);
+
```

Comment:
I guess you'll add a FlashMessage with another PR?

Reply:
Yes, exactly. I added #875 to track this.